### ISSUE TITLE
SI_CONVERSION case sensitivity

### ISSUE THREAD

**Issue by Alrobbertz** [IHDE-Alliance/ISTP_metadata#7]
The SPDF ISTP metadata guide lists `SI_conversion` while this repository lists `SI_CONVERSION`. This this attribute case-sensitive? If so, which case should we use? 

Reference: 
https://spdf.gsfc.nasa.gov/istp_guide/vattributes.html#variable

**Comment by candeynasa** [comment]
The metadata is supposed to be case-sensitive, but in practice most SPDF software ignores case, and I expect this is true of other software like Autoplot.org.  I had to research this attribute.  I see the original Cluster CDF metadata documents like https://caa.esac.esa.int/documents/DS_QMW_TN_0003_v1.6.pdf use SI_conversion, while the CEF documents https://caa.esac.esa.int/documents/CAA-MDD-0001_v35.pdf and https://caa.esac.esa.int/documents/DS-QMW-TN-0010.pdf went to upper case. The CDF one also adds SI_conversion_ptr for pointing to a dimensional string variable for cases where there are varying units in the data variable (similar to UNIT_PTR). I looked at the occurrences in the metadata at CDAWeb and found 28878 using SI_conversion, 16752 using SI_CONVERSION, 8517 using SI_conv, 4 using SI_CONV, and 8 using SI_conversele.  Even though SI_conversion is most common, perhaps we should adopt SI_CONVERSION to match the later Cluster and Solar Orbiter https://s2e2.cosmos.esa.int/confluence/display/SOSP/Metadata+Definition+for+Solar+Orbiter+Science+Data standards.
